src/rcore.c

```diff
@@ -6075,6 +6090,8 @@ static int32_t AndroidInputCallback(struct android_app *app, AInputEvent *event)
         }
         else CORE.Input.Keyboard.currentKeyState[keycode] = 0;  // Key up
 
+        if (AKeyEvent_getAction(event) == AKEY_EVENT_ACTION_MULTIPLE) CORE.Input.Keyboard.keyRepeatInFrame[keycode] = 1;
```

Comment:
it seems that presently when we get `AKEY_EVENT_ACTION_MULTIPLE` we mark it as keyup.
```suggestion
const eventAction = AKeyEvent_getAction(event);
// handle each AKEY_EVENT_ACTION_DOWN, AKEY_EVENT_ACTION_UP, AKEY_EVENT_ACTION_MULTIPLE
```